static/chat/js/gui.js

```diff
@@ -468,6 +468,14 @@
                 return;
             }
 
+            // parse backlog preliminarily to initialize nicks so the formatter will mark them as such (clickable)
+            for (var i = 0, j = this.backlog.length; i < j; i++){
+                obj = JSON.parse(this.backlog[i].substring(this.backlog[i].indexOf(' ')));

```

Comment:
same global issue, plus we are parsing the backlog twice now, should centralize it possibly in chat.js